### DISCUSSION TITLE
Cancel Whatsapp schedule

### DISCUSSION THREAD

**Discussion by newfaboc** [kirimwa-id/forum#69] [Tanya Jawab] [answered]
Hi rio,
Saya telah mencoba fitur schedule send Whatsapp. 
Saya aingin bertanya : Apakah ada cara untuk membatalkan "Sent at" dalam api whatsapp yang terlanjur terexecute, namun Belum waktunya di kirim oleh api whatsapp ?

Terima kasih

**Comment by rioastamal** [comment, marked as answer]
Halo @newfaboc, saat ini tidak ada fitur untuk membatalkan pesan yang sudah masuk antrian pada API KirimWA.id.
